.changeset/two-walls-fetch.md

```diff
@@ -0,0 +1,9 @@
+---
+"@comet/cms-admin": minor
+"@comet/cms-site": minor
+"@comet/cms-api": minor
+---
+
+Add preview image to `YouTubeVideoBlock` and `DamVideoBlock`
+
+The `YouTubeVideoBlock` and the `DamVideoBlock` do now support a preview image out of the box. For customisation the default `VideoPreviewImage` component can be overridden with the optional `renderPreviewImage` method.
```

Comment:
As discussed, please recommend here to use the blocks from the library in the site instead of custom ones defined in the application.